packages/daemon/src/host.js

```diff
@@ -68,10 +69,32 @@ export const makeHostMaker = ({
       terminator,
     });
 
+    /**
+     * @param {any} newController
```

Comment:
I believe there's an importable controller type in `types.d.ts`.

Reply:
cool, i have a few type improvements ill add